parquet-column/src/main/java/org/apache/parquet/column/statistics/BinaryStatistics.java

```diff
@@ -136,4 +154,9 @@ public void setMinMax(Binary min, Binary max) {
     this.min = min;
     this.markAsNotEmpty();
   }
+
+  @Override
+  public BinaryStatistics copy() {
+    return new BinaryStatistics(this);
```

Comment:
The docs for this method state that "all the values are copied" but the copy constructor doesn't call `Binary.copy()`. I don't think it is necessary for it to copy, so you should remove the part about copying values from the docs.